openpype/hosts/resolve/api/plugin.py

```diff
@@ -410,13 +423,18 @@ def load(self):
             source_out -= handle_end
 
         # include handles
-        if self.with_handles:
-            source_in -= handle_start
-            source_out += handle_end
+        if not self.with_handles:
```

Comment:
Could it be a situation where line 421 and this one to be true, so we add /substract twice the handles?

Reply:
good point, better to merge both those conditions